packages/core/src/tracing/dynamicSamplingContext.ts

```diff
@@ -19,6 +19,7 @@ export function getDynamicSamplingContextFromClient(
   const options = client.getOptions();
 
   const { publicKey: public_key } = client.getDsn() || {};
+  // TODO(v8): Remove segment from User
```

Comment:
Nice, that means we won't need the scope anymore for the dsc, right? Cc @Lms24 we can simplify this a bit in V8 then 🎉